packages/react/src/components/UIShell/HeaderContainer.js

```diff
@@ -1,42 +0,0 @@
-/**
```

Comment:
I used `git mv` on this file, but after my last push git considers it as deleted. I guess the file is different enough now that it considers it as deleted/new rather than renamed?

Reply:
I wonder if removing the large example in the comment will revert it back to `git mv` diff? Its fine either way 👍🏻

Reply:
It did! 😄